### PR TITLE
tweak documentation

### PR DESCRIPTION
**Pull request by SamVerschueren**
Tweaked the documentation a little bit more to your standard. Close if you like the old documentation :).

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -13,33 +13,38 @@ $ npm install --save map-obj
...
-  An empty object will be created if not supplied.
- 
-**Returns:** `target`
+### mapObj(source, mapFn, [target])
+
+#### source
+
+Type: `object`
+
+The source object to copy properties from.
```
Comment (sindresorhus) [inline readme.md]: Drop the `the`.

File: readme.md [hunk 2]
```diff
@@ -13,33 +13,38 @@ $ npm install --save map-obj
...
+
+#### mapFn
+
+Type: `function`
+
+The mapping function.
```
Comment (sindresorhus) [inline readme.md]: Drop the `the`.

File: readme.md [hunk 3]
```diff
@@ -13,33 +13,38 @@ $ npm install --save map-obj
...
+
+- It has signature `mapFn(sourceKey, sourceValue, source)`. 
+- It must return a two item array: `[targetKey, targetValue]`.
+
+#### target
+
+Type: `object`  
+Default: `{}`
+
+The target object to map properties on to.
```
Comment (sindresorhus) [inline readme.md]: Drop the `the`.

File: readme.md [hunk 4]
```diff
@@ -13,33 +13,38 @@ $ npm install --save map-obj
...
 ```js
-var mapObj = require('map-obj');
+const mapObj = require('map-obj');
 
-var newObject = mapObj({foo: 'bar'}, function (key, value, object) {
-	// first element is the new key and second is the new value
-	// here we reverse the order
-	return [value, key];
-});
+const newObject = mapObj({foo: 'bar'}, (key, value) => [value, key]);
```
Comment (SamVerschueren) [inline readme.md]: Not sure about dropping the comments here?

**Comment by sindresorhus** [comment]
Goodie! It actually was somewhere in my todo list to do this at some point.

**Pull request closed without merging**